src/controllers/Flashcards.js

```diff
@@ -70,11 +78,24 @@ const getDeckWithFlashcards = async (req, res) => {
     }
 };
 
-// (2) get the flashcards that appertain to a specific deck 
+// (2) get the flashcards that appertain to a specific deck - not used for the moment (with no detailed information about each flashcard)
 const getFlashcardsForDeck = async (req, res) => {
-    req.body.createdBy = req.user.userId;
-    const flashcards = await Flashcard.find({ createdBy: req.user.userId, deck: req.id }).sort('createdAt');
-    res.status(StatusCodes.CREATED).json({ flashcards, count: flashcards.length });
+    try {
+        req.body.createdBy = req.user.userId;
```

Comment:
[Requests Should Generally Be Immutable]
This doesn't seem to do anything? Can we remove this line? What was the intention?

Reply:
There is no route for this controller either, as the previous controller does a similar job and I had an issue adding a separate route for this controller. So, I will remove the entire controller from here.